assets/styles/global/_button.scss

```diff
@@ -102,9 +102,12 @@ button,
 .btn.disabled,
 .btn[disabled],
 fieldset[disabled] .btn {
-  @extend .bg-disabled;
   cursor: not-allowed;
-  border: solid thin var(--input-disabled-border);
+  color: var(--disabled-text) !important;
```

Comment:
These changes add a dark border to the disabled `Import Yaml` button in the cluster explorer header

Reply:
Adding a greater degree of specificity to the styling in Header.vue fixes this: 
<img width="507" alt="Screen Shot 2021-02-23 at 11 09 25 AM" src="https://user-images.githubusercontent.com/42977925/108887661-c476e500-75c7-11eb-9f62-7f112caaf818.png">